pyscriptjs/src/stdio.ts

```diff
@@ -36,6 +38,42 @@ export class CaptureStdio implements Stdio {
     }
 }
 
+/** Stdio provider for sending output to DOM element
+ *  specified by ID. Used with "output" keyword.
+ *
+ */
+export class TargetedStdio implements Stdio{
+
+    target_id: string;
+
+    constructor(target_id: string) {
+        this.target_id = target_id;
+    }
+
+    stdout_writeline (msg: string) {
+        // Elements are identified by ID at write-time, not
+        // when constructed, in case the running script changes element IDs
+        const target = document.getElementById(this.target_id)
+        console.log(target)
```

Comment:
Is this a debug log? otherwise we might want to replace it with `logger.info` instead

Reply:
Ah, a leftover debug log. Deleted.